layers/meta-resin-intel-quark/recipes-devtools/go/files/0001-Revert-runtime-check-and-fail-early-with-a-message-i.patch

```diff
@@ -6,7 +6,7 @@ Subject: [PATCH 1/2] Revert "runtime: check and fail early with a message if
 
 This reverts commit 35e84546d7c3cc599dd5bf31ad86bf8d3d841d04.
 
-Upstream Status: Inappropriate
+Upstream Status: Inappropriate [revert]
```

Comment:
That is not very informative though. Here are some examples: http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations